Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
They mean the same thing, ```:key => value``` can be abbreviated as ``key: value`` in Ruby.
By the way, I copied this line from https://github.com/github/pages-gem.